src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -14,12 +14,35 @@
 import java.util.stream.Collectors;
 import uk.ac.ebi.embl.api.entry.Entry;
 import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.api.entry.location.CompoundLocation;
+import uk.ac.ebi.embl.api.entry.location.Location;
 import uk.ac.ebi.embl.api.entry.qualifier.Qualifier;
+import uk.ac.ebi.embl.api.entry.sequence.Sequence;
 import uk.ac.ebi.embl.converter.IConversionRule;
 import uk.ac.ebi.embl.converter.gff3.*;
 import uk.ac.ebi.embl.converter.utils.ConversionEntry;
 import uk.ac.ebi.embl.converter.utils.ConversionUtils;
 
+enum CircularFeatures {
```

Comment:
Please move this ENUM to the bottom of the class; it is more readable when the main Class of the Java file is in the beginning of the file